angular/tech-show-frontend/src/app/components/login/login.component.html

```diff
@@ -1,10 +1,12 @@
 <div class="login-container">
-  <div class="login-form-container">
+  <form class="login-form-container" [formGroup]="loginForm" (ngSubmit)="onSubmit()">
     <h1 class="login-title">Login</h1>
     <label htmlFor="email" class="login-input-title">Email</label>
-    <input id="email" class="login-input" />
+    <input id="email" type="email" class="login-input" formControlName="email"/>
+    <span class="input-error" [style.visibility]="email.invalid ? 'visible' : 'hidden'">El campo debe contener un email válido</span>
     <label title="password">Password</label>
-    <input id="email" class="login-input-title" class="login-input" />
-    <button type="button" class="login-button">Login</button>
-  </div>
+    <input id="email" type="password" class="login-input-title" class="login-input" formControlName="password" />
+    <span class="input-error" [style.visibility]="password.invalid ? 'visible' : 'hidden'">La contraseña debe tener al menos 6 caractéres</span>
```

Comment:
you can change this for an ngIf instead

Reply:
ngIf removes the element from the DOM, this setst the visibility so the element is still in the dom but not visible. That's why I did it this way.